localstack/services/apigateway/helpers.py

```diff
@@ -604,6 +605,33 @@ def import_api_from_openapi_spec(
     # Remove default root, then add paths from API spec
     rest_api.resources = {}
 
+    def create_authorizer(path_payload: dict) -> Authorizer:
+        if "security" not in path_payload:
+            return None
+
+        security_schemes = path_payload.get("security")
+        for security_scheme in security_schemes:
+            for security_scheme_name, _ in security_scheme.items():
+                if security_scheme_name in body.get("securityDefinitions"):
+                    security_config = body.get("securityDefinitions", {}).get(security_scheme_name)
+                    aws_apigateway_authorizer = security_config.get(
+                        "x-amazon-apigateway-authorizer"
+                    )
+                    return rest_api.create_authorizer(
+                        create_resource_id(),
+                        name=security_scheme_name,
+                        authorizer_type=aws_apigateway_authorizer.get("type"),
+                        provider_arns=None,
+                        auth_type=security_config.get("x-amazon-apigateway-authtype"),
+                        authorizer_uri=aws_apigateway_authorizer.get("authorizerUri"),
+                        authorizer_credentials=aws_apigateway_authorizer.get(
+                            "authorizerCredentials"
+                        ),
+                        identity_source=aws_apigateway_authorizer.get("identitySource"),
+                        identiy_validation_expression=None,
+                        authorizer_result_ttl=300,
```

Comment:
nit: we could use `aws_apigateway_authorizer.get("authorizerResultTtlInSeconds")` here, and fall back to 300 as default value.